### DISCUSSION TITLE
Melting layer detection

### DISCUSSION THREAD

**Discussion by rwall2** [ARM-DOE/pyart#1410] [Q&A] [answered]
I see that version 1.6.4 has retrieve.melting_layer_giangrande, but version 1.12.7 doesn't seem to have that function available. Was there a reason it was dropped? Or does it still exist and I can't find it?

**Comment by zssherman** [comment, marked as answer]
@rwall2 The function was added to the Meteoswiss fork of Py-ART version 1.6.4 by their members there and developed that version to fit the Meteoswiss Pyrad etc needs at the time.  This was the reason why it was never added to the ARM-DOE Py-ART version as they developed that fork almost independently of the original repository. We've had the discussion in the past to do PRs for some code between the two versions, but has been on hold at the moment. Hope that helps answer your question!

**Reply by rwall2** [reply]
Thanks! Is there a method to find the melting layer in version 1.12?

**Reply by zssherman** [reply]
@rwall2 I don't believe there is. I'll double check though! If we don't, I think it would be a good addition to 1.12, but at the moment it would be low priority sadly.
